src/develop/ui/patterns/web/navigation/wizard.md

```diff
@@ -3,153 +3,108 @@ tags: runtime-traditionalweb;
 
-Split complex tasks and process into steps.
+**How to Use the Wizard UI Pattern**
 
-Use the Wizard to present numbered steps or conditions that the end user needs to complete or reach a goal. Use it to split large tasks into more manageable steps. Wizards usually include explicit button navigation to move a step forward or backward. 
+1. In Service Studio, in the Toolbox, search for  `Wizard`. 
 
-**How to use**
+    The Wizard widget is displayed.
+    
+     ![](<images/wizard-image-23.png>)
+
+1. From the Toolbox, drag the Wizard widget onto your application's screen.
 
-1. Drag Wizard pattern into the preview.
+    By default, the Wizard widget contains three Wizard Item blocks. Each Wizard Item represents a step. You can add or delete Wizard Items as required.
 
     ![](<images/wizard-image-1.png>)
 
-1. Drag as many WizardItems in the Content placeholder as required. 
+**Create a Wizard with Navigation Buttons**
 
-1. Set the content in the placeholders.
+The following example demonstrates how you can create a three step Wizard with navigation buttons.
 
-1. Set the mandatory values.
+1. Rename each of the Wizard Items to the following: 
+    * Shopping Details
+    * Payment Details
+    * Review Order
 
-    ![](<images/wizard-image-2.png>)
+        ![](images/wizard-image-14.png)
 
-1. Publish and test.
+1. Create an Input Parameter with the following properties:
+    
+    ![](images/wizard-image-13.png)
+ 
+    This input paramenter controls the current Wizard step. 
+    Setting the **Default Value** to 1 ensures the Wizard always starts at step 1.
 
+1. From the Toolbox, drag two Button widgets onto your screen and set their names to **Previous** and **Next**.
 
-## Input Parameters
+1. Define the behaviour for each of the navigation buttons. 
 
-### Wizard
+      ![](images/wizard-image-15.png)
 
-| **Input Name** |  **Description** |  **Type** | **Mandatory** | **Default Value** |
-|---|---|---|---|---|
-| Orientation  |  Set the orientation. | Orientation Identifier | False | Entities.Orientation.Horizontal |
-| ExtendedClass  |  Add custom style classes to this Block. | Text | False | none |
+1. Create an expression that defines the status of each of the Wizard Items (current step, step already completed, step yet to be completed), depending on what stage the user is at.
 
-### Wizard Item
-
-| **Input Name** |  **Description** |  **Type** | **Mandatory** | **Default Value** |
-|---|---|---|---|---|
-| Step  | Set the step. | Step Identifier | True | none |
-| UseTopLabel  |  If true, label is placed above the icon. |  Boolean | False | False |
-| ExtendedClass  |  Add custom style classes to this Block. | Text | False | none |
-  
-## Layout and Classes
+    ![](images/wizard-image-16.png)
 
-![](images/wizard-image-3.png?width=750)
+1. Create a condition for both the **Previous** and **Next** buttons. This ensures that the user can never goes above or below the actual number of steps in the Wizard.
```

Comment:
@annettewilliamsoutsystems @MiguelDomingues Same here - I infer that this would be in an action, but we haven't mentioned it until now? If not, where do we add this "condition" - that seems to be an actual If node?

Reply:
Edited

1. To create a condition that only displays the **Previous** button when applicable, ensuring that the user can never go below the number of steps in the Wizard, select the **Previous** button, right-click, and select **Enclose in If**.

    ![](images/wizard-image-26.png) 

1. On the Properties tab, in the **Condition** property, enter the following:

   `CurrentStep > 1`

    ![](images/wizard-image-17.png)

       
   Repeat this step for the **Next** button. In this example, we use `CurrentStep < 4` for the **Next** button.